tm2/pkg/std/coin_benchmark_test.go

```diff
@@ -2,24 +2,30 @@ package std
 
 import (
 	"fmt"
-	"strconv"
 	"testing"
 )
 
 func BenchmarkCoinsAdditionIntersect(b *testing.B) {
-	b.Skip("TODO: panicking benchmark")
 	benchmarkingFunc := func(numCoinsA int, numCoinsB int) func(b *testing.B) {
 		return func(b *testing.B) {
 			b.Helper()
 
 			coinsA := Coins(make([]Coin, numCoinsA))
 			coinsB := Coins(make([]Coin, numCoinsB))
 
+			maxCoins := numCoinsA
+			if numCoinsB > numCoinsA {
+				maxCoins = numCoinsB
+			}
+			denomLength := len(fmt.Sprint(maxCoins))
```

Comment:
can use built-in `max` function

Reply:
ef2bdb5b6df7ee63140626fa78b4923101780aa9